docs/Features.md

```diff
@@ -70,6 +70,7 @@ import { useDataProvider } from 'react-admin';
 const PostList = () => {
     const [posts, setPosts] = useState();
     const [error, setError] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
#9285 also changed the default values for the `posts` and `error` states, and they made more sense there IMO.

I think we should also include this change.

```suggestion
    const [posts, setPosts] = useState([]);
    const [error, setError] = useState();
    const [isLoading, setIsLoading] = useState(true);
```